src/components/Identity/util.js

```diff
@@ -0,0 +1,61 @@
+import { AUTH_STATES } from "./constants";
+import isObject from "../../utils/isObject";
+
+const ERROR_MESSAGE = "Invalid customer Id format.";
```

Comment:
When we're writing in English, let's use `ID`. The capitalization of `*Id` or `id` is just to fit camelcasing rules (as we perceive the rules).